### PR TITLE
"alt" update in problem-199 "iterative circle packing"

### LINKED ISSUE #44245: Mistake in the alt text for the diagram in "Problem 199: Iterative Circle Packing"
In the challenge below, the alt text for the diagram is set to "a diagram of non-overlapping concentric circles".
But if my understanding is correct, they are not "concentric circles" (circles which share the same center).

## Affected page

Coding Interview Prep > Project Euler > Problem 199: Iterative Circle Packing
https://www.freecodecamp.org/learn/coding-interview-prep/project-euler/problem-199-iterative-circle-packing

## Expected behavior

The alt text should be changed to "a diagram of non-overlapping circles" or something.
(I'm not a native English speaker, so please fix it if there is a better description.)

## Screenshots

project_euler_199_issue

## System

- Device: Laptop
- OS: Windows 10
- Browser: chrome
- Version: 96.0.4664.45

## Additional context

Originally reported in a comment on Crowdin duing translation.
https://freecodecamp.crowdin.com/translate/690087755819f2f67b26dbee3be0eb53/22524/en-ja/132?filter=basic&value=7#q=662762

### PR DESCRIPTION
**Pull request by shashank2906**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #44245 

Problem - 199 "Iterative Circle Packing"
Updated the "alt" attribute of the image in the problem statement from "a diagram of non-overlapping concentric circles"  to "a diagram of non-overlapping circles" as those are not concentric circles.

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me!

**Comment by shashank2906** [comment]
Is something else required in the changes made.?

**Comment by nhcarrigan** [comment]
This PR is waiting for a second review. Our workflow requires two approvals before a contribution is merged. Thanks for your patience. 🙂

**Review by moT01 — APPROVED** [review]
LGTM @shashank2906 :tada: Congratulations on your first contribution to this freeCodeCamp repo :tada:

**Comment by shashank2906** [comment]
Thank you so 😃

**Comment by shashank2906** [comment]
Thank you so much 😇

**Pull request merged**
